.github/workflows/robustness_nightly.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
     with:
       count: 100
       testTimeout: 200m
-      runs-on: "['ubuntu-latest-8-cores']"
```

Comment:
REF: https://github.com/etcd-io/bbolt/pull/612

The original motivation is not for nested virt. The free-tier runner has limitation. It seems that there was runner agent monitoring memory or cpu usage. If the resource usage is higher than expected, the agent will kill the process https://github.com/actions/runner-images/issues/6680#issuecomment-1335778010

By default, the free-tier runner only has 3 cores. I'm not sure that it can do nested virt.

Reply:
The free tier has been upgraded :+1: They are 4 core with 16gb of ram now.

Reply:
Thanks for the information 👍 

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories